docs/helm/README.md

```diff
@@ -24,7 +24,37 @@
 
   * **Code:** 403 BAD REQUEST <br />
     **Content:** `{ error : "error message" }`
-    
+
+
+**Get Helm Release**
+----
+  _Returns installed helm release in JSON_
+
+* **URL**
+
+     `/api/helm/release`
+
+* **Method:**
+  
+  `GET` 
+  
+*  **URL Params**
+
+   **Optional:**
+ 
+   `ns=[string]` - Namespace
+   
+   `release_name=[string]` - Helm Release Name
+
+* **Success Response:**
+  
+  * **Code:** 200 <br />
+ 
+* **Error Response:**
+
+  * **Code:** 403 BAD REQUEST <br />
```

Comment:
403 is "Forbidden", 400 is "Bad Request". But it looks like we're actually returning "Bad Gateway"?